components/fieldgroup/stories/fieldgroup.stories.js

```diff
@@ -21,6 +20,10 @@ export default {
 	title: "Field group",
 	component: "FieldGroup",
 	argTypes: {
+		label: {
+			name: "Label",
```

Comment:
Oh nice addition. The default value for `label` is "Field group label." We could add that here to either the `name`, or we could add a `description`.

```
label: {
	name: "Label",
        description: "The field label for the entire field group",
        ...
```

And you could probably come up with a better description than me right now 😁